apps/nestjs/src/main.ts

```diff
@@ -9,18 +9,13 @@ async function bootstrap() {
   Sentry.init({
     environment: 'qa', // dynamic sampling bias to keep transactions
     dsn: process.env.SENTRY_DSN,
-    includeLocalVariables: true,
     tunnel: `http://localhost:3031/`, // proxy server
     tracesSampleRate: 1,
   });
 
   const app = await NestFactory.create(AppModule);
-
-  app.use(Sentry.Handlers.requestHandler());
-  app.use(Sentry.Handlers.tracingHandler());
-
-  // Uncaught exceptions are not sent as event in v7, in v8 they are sent as events and transactions
-  app.use(Sentry.Handlers.errorHandler());
+  // @ts-ignore
+  Sentry.setupNestErrorHandler(app);
```

Comment:
The type for the `app` param does not seem quite right. 
![image](https://github.com/getsentry/sentry-javascript-examples/assets/32902192/0b96ebef-a1c6-446f-9906-540e52029145)